### PR TITLE
Docs/influx fix

### PR DESCRIPTION
**Pull request by ocket8888**
## What does this PR (Pull Request) do?

- [x] This PR is not related to any Issue

This fixes an issue where Influx connections were defaulting to secure, breaking existing installs. It also properly documents the various Traffic Ops configuration files (to the best of my knowledge) to make their behavior explicit and clear.

## Which Traffic Control components are affected by this PR?
- Documentation
- Traffic Ops

## What is the best way to verify this PR?
Mostly just build and read documentation. Also build an ATC environment with minimally Traffic Ops and Traffic Stats - specifically Traffic Stats's InfluxDB clusters should not be configured to use SSL -, request `deliveryservice_stats` (properly) and observe that it works

## The following criteria are ALL met by this PR
- [x] Tests are unnecessary
- [x] This PR includes documentation
- [x] An update to CHANGELOG.md is not necessary
- [x] This PR includes any and all required license headers
- [x] This PR does not include a database migration
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY**

### CODE REVIEW

File: docs/source/admin/traffic_ops.rst [hunk 1]
```diff
@@ -0,0 +1,706 @@
...
+
+	.. code-block:: shell
+		:caption: Starting PostgreSQL with :manpage:`systemd(1)`
+
+		systemctl enable postgresql-9.6
+		systemctl start postgresql-9.6
+		systemctl status postgresql-9.6 # Prints the status of the PostgreSQL service, to prove it's running
+
+
+#. Build a :file:`traffic_ops-{version string}.rpm` file using the instructions under the :ref:`dev-building` page - or simply download a pre-built `release <https://github.com/apache/trafficcontrol/releases>`_.
```
Comment (mhoppa) [inline docs/source/admin/traffic_ops.rst]: I dont see any pre-built rpms attached to the releases which is actually very surprising.

File: docs/source/admin/traffic_ops.rst [hunk 2]
```diff
@@ -0,0 +1,706 @@
...
+
+	.. caution:: If Traffic Ops is unable to connect to Traffic Vault, many of its core features will not function. In particular, it will be impossible to create :term:`Delivery Services` that use HTTPS or DNSSEC.
+
+	.. impl-detail:: The name of this field is derived from the current database used in the implementation of Traffic Vault - `Riak KV <https://riak.com/products/riak-kv/index.html>`_.
+
+:secrets: This is an array of strings, which cannot be empty. The first secret in the array is used to encrypt Traffic Ops authentication cookies - multiple Traffic Ops instances serving the same CDN need to share secrets in order for users logged into one to be able to use their cookie as authentication with other instances.
+:smtp:    This optional section contains options for connecting to and authenticating with an :abbr:`SMTP (Simple Mail Transfer Protocol)` server for sending emails. If this section is undefined (or if ``enabled`` is explicitly ``false``), Traffic Ops will not be able to send emails and certain :ref:`to-api` endpoints that depend on that functionality will fail to operate.
```
Comment (mhoppa) [inline docs/source/admin/traffic_ops.rst]: wasnt this added for `.. versionadded:: 4.0` like the influxdb configuration?

Comment (ocket8888) [reply docs/source/admin/traffic_ops.rst]: yes, yes it was

File: docs/source/admin/traffic_ops.rst (hunk 1 above)
Comment (ocket8888) [reply docs/source/admin/traffic_ops.rst]: That is odd. I know we had downloads for those somewhere, but I can't find them.

File: docs/source/admin/traffic_ops.rst [hunk 3]
```diff
@@ -0,0 +1,708 @@
...
+	:pid_file:          When the `Legacy Perl Script`_ starts as a daemon, it will write its :abbr:`PID (Process ID)` to this file - so be sure that ``user``/``group`` can properly access it! The default setting is :file:`/var/run/traffic_ops.pid`.
+	:user:              Sets the UNIX user as whom the `Legacy Perl Script`_ will run. By default this should be the ``trafops`` user who is created when the Traffic Ops RPM is installed.
+	:workers:           This is used only by the `Legacy Perl Script`_, and has no known purpose. Though, one could guess that it's the number of concurrent HTTP server workers allowed.
```
Comment (rawlinp) [inline docs/source/admin/traffic_ops.rst]: I believe this is the number of Mojolicious worker processes that will be spawned to handle HTTP requests for TO-Perl.

**Pull request merged**